lib/scsi_defs.h

```diff
@@ -0,0 +1,121 @@
+#ifndef LIB_SCSI_DEFS_H
+#define LIB_SCSI_DEFS_H
+
+namespace scsi_defs {
+
+  enum Status {
```

Comment:
Use enum class. https://abseil.io/tips/86 
Also, the advice in the style says to use kGood, kCheckCondition etc style instead of ALL_CAPS.